networktest.go

```diff
@@ -177,9 +190,30 @@ func (l *lightningNode) Start(lndError chan error) error {
 
 	l.cmd = exec.Command("lnd", args...)
 
-	// Redirect stderr output to buffer
+	outWriters := []io.Writer{
+		&l.stdout,
+		&l.multi,
+	}
+
 	var errb bytes.Buffer
-	l.cmd.Stderr = &errb
+	errWriters := []io.Writer{
+		&l.stderr,
+		&l.multi,
+		&errb,
+	}
+
+	if l.name == "satoshis" {
+		fileName := fmt.Sprintf("/tmp/%v.txt", l.name)
+		f, err := os.OpenFile(fileName, os.O_WRONLY|os.O_CREATE, os.ModePerm)
```

Comment:
File in temporary directory will be removed at the next computer reboot, `afaict` this is what `tmp` directory was designed for.